Rakefile

```diff
@@ -56,6 +57,17 @@ task :run, :command do |t, args|
   run_command args[:command]
 end
 
+desc "Updates the rspec.github.io docs"
+task :update_docs, [:version, :branch, :website_path] do |t, args|
+  args.with_defaults(:website_path => "../rspec.github.io")
```

Comment:
Nope, this is good, I use rspec-dev for things like this and just let it do it's thing, I just develop elsewhere.